### PR TITLE
Disable shuffling of data during evaluation

### LINKED ISSUE #671: Make Order of Evaluation Examples Consistent
Currently the [call](https://github.com/twosixlabs/armory/blob/master/armory/data/datasets.py#L235) to `tfds.load()` is by default [set to True](https://github.com/twosixlabs/armory/blob/master/armory/data/datasets.py#L191).

It appears that a seed is only being set for the poisoning scenario, so for other scenarios we have the following issue: when we load in benign examples for evaluation, and then subsequently we load in examples to perturb before evaluating on, the ordering in these cases doesn't align.

### PR DESCRIPTION
**Pull request by lcadalzo**
fixes #671

### CODE REVIEW

**Comment by dxoigmn** [comment]
FYI: When we have set `shuffle_files=False`, we have noticed memory leaks. This doesn't matter so much if you have swap on, but on my machine with 126GB of memory and swap off, the process gets killed by the kernel due to OOM. It may be worth running this and watching memory usage as the UCF101 scenario progresses.

**Comment by davidslater** [comment]
@dxoigmn This is separate from the other UCF101 memory issues related to pytorch + attacks?

**Comment by dxoigmn** [comment]
@davidslater Yes. This is a CPU memory leak.

**Comment by davidslater** [comment]
We haven't been able to duplicate a memory leak on Ubuntu 18.04 on the UCF101 scenario. We'll keep this in mind though, if it pops up again.

**Comment by dxoigmn** [comment]
> We haven't been able to duplicate a memory leak on Ubuntu 18.04 on the UCF101 scenario. We'll keep this in mind though, if it pops up again.

Interesting! When we upgrade Armory I will try to replicate on our side and report back.

**Comment by davidslater** [comment]
Thanks, I appreciate it!

**Comment by dxoigmn** [comment]
Quick Q: When the memory-leak test was done, was it run in Docker?

I don't see the leak happen in the Armory Docker containers. I do see it happen when running outside of Docker, so it may have something to do with the PyPI version of Tensorflow I install on the host versus the Conda version installed in the Docker container.

**Comment by davidslater** [comment]
Yes, we were testing it in Docker. I think we also use the conda installed version when running on host, so if that is the issue, we wouldn't have found it.
